app.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * If you would like to turn your application into a standalone executable, look at server.js file
+ *
+ */
+
+import path from 'node:path'
+import AutoLoad from '@fastify/autoload'
+import swagger from './swagger.js'
+
+export default async function app (fastify, opts) {
+  fastify.register(swagger)
```

Comment:
OK

Reply:
The doc shows an example with await effectively, I had missed it.

Reply:
Yes, so to extend on this. I had issues with under-pressure plugin when the plugin was loaded synchronize together with my database, causing my `healthCheck` (part of under pressure plugin) to fail during start-up and I needed to wait several seconds. To fix this I added `awaiy` in front of all my `register` methods. As well as the async method I used too bootstrap my app. Which eventually fixed all the synchronize issues. Now my application starts-up within a second, and is fully functional even with the under-pressure with a db healthCheck configured. 

I just wanted to mention this, so you understand my full story.

Reply:
That's interesting, we may have similar problems in this demo, and this would be an opportunity to think about it and possibly comment on it.

Reply:
Are you allowed to share the code or at least an example that illustrates the problem? That might be useful.
Also, I was wondering if you only configure for the production environment or if you also have test cases?

Reply:
I'm afraid that code I do not have anymore, after all I fixed it  :D. After you completed this demo PR, I can of course test if I notice any issue with under-pressure, but as long as you do not setup a DB config with a heathcheck, this will not happen.